app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id] if session[:user_id]
```

Comment:
thế nếu ko có thì sao

Reply:
Trả về người dùng đã đăng nhập hiện tại (nếu có). ý anh hỏi là nếu k có người dùng đăng nhập thì nó sẽ như thế nào hay là sao ạ?? e chưa hiểu rõ ạ?? =)))

Reply:
ý là toán tử ||= ở đây có ý nghĩa gì?

Reply:
toán tử ||= nghĩa là bằng hoặc bằng ạ.
đoạn code này, nếu user_id = session[:user_id] thì trả về @current_user, nếu @current_user thì trả về @current_user = User.find_by id: user_id